nanopub/trustyuri/CheckFile.py

```diff
@@ -0,0 +1,35 @@
+import codecs
+import logging
+import sys
+
+from nanopub.trustyuri import ModuleDirectory, TrustyUriUtils
+from nanopub.trustyuri.TrustyUriResource import TrustyUriResource
+
+try:
+    from urllib2 import urlopen
+except Exception:
+    from urllib.request import urlopen
+
+
+def check(args):
+    filename = args[0]
```

Comment:
I suppose this could be more robust with checks on the commandline params?

Reply:
Not sure, it's a leftover from the original `trustyuri` but I don't use this part so I did not updated it

Ideally we should use the same library `typer` that we use for the rest of the CLI, and create a new CLI entrypoint or add a `np trusty` group of commands to the existing CLI